### PR TITLE
No map? Let's not crash

### LINKED ISSUE #1782: Crash on print when having no map
When having a print layout with a map item inside it works.
When having a print layout with only a label inside it crashes.

I'll provide more information.

It's 1.9.

### PR DESCRIPTION
**Pull request by nirvn**
Fixes #1782 .

### CODE REVIEW

**Review by signedav — APPROVED** [review]
Perfect

File: src/core/qgismobileapp.cpp [hunk 1]
```diff
@@ -957,7 +957,8 @@ bool QgisMobileapp::print( const QString &layoutName )
   if ( !layoutToPrint || layoutToPrint->pageCollection()->pageCount() == 0 )
     return false;
 
-  layoutToPrint->referenceMap()->zoomToExtent( mMapCanvas->mapSettings()->visibleExtent() );
+  if ( layoutToPrint->referenceMap() )
+    layoutToPrint->referenceMap()->zoomToExtent( mMapCanvas->mapSettings()->visibleExtent() );
   layoutToPrint->refresh();
```
Comment (signedav) [inline src/core/qgismobileapp.cpp:962]: Do we need to refresh or can we take that into the if?

Comment (nirvn) [reply src/core/qgismobileapp.cpp:962]: To be safe, I think we should -- refresh will also refresh non-map layout items. Tables, HTML, expression-based labels, etc.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 7a9fbc2f1c5d54778bd646c1de500eb3b9c90ed9:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1783-b6adf7adef22f5318053fa21205f40032bb4ebdc-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1783-b6adf7adef22f5318053fa21205f40032bb4ebdc-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1783-b6adf7adef22f5318053fa21205f40032bb4ebdc-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-1783-b6adf7adef22f5318053fa21205f40032bb4ebdc-x86.apk)

**Pull request merged**